src/components/button/bl-button.ts

```diff
@@ -104,6 +104,13 @@ export default class BlButton extends LitElement {
   @property({ type: Boolean, reflect: true })
   autofocus = false;
 
+
+  /**
+   * Sets the associated form of the button. Use when `type` is set to `submit` and button is not inside the target form.
+   */
+  @property({ type: String })
+  form: HTMLFormElement | string;
```

Comment:
How to pass an element here? By setting property?

Reply:
You can either pass the id of the form element (via attribute or property) or the DOM reference of it (by property). Both would work.

I gave some examples in the storybook docs.